tests/providers/test_mailgun.py

```diff
@@ -83,6 +83,10 @@ def test_mailgun_error_response(self, provider):
         data = {
             'api_key': 'FOO',
             'message': 'bla',
+            'to': 'BAR',
```

Comment:
No need for these. Some test can run run by Travis due to secret env vars which can only be access by me. Just run `pytest -m "not online"` and those are the ones that should pass (other than the stuff you added)